<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>musicPlayer</class>
 <widget class="QMainWindow" name="musicPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>musicPlayer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonLast">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>523</y>
      <width>110</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(202, 248, 255,125);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/image/last.png</normaloff>:/image/last.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPause">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>523</y>
      <width>110</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(187, 184, 255,125);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/image/star.ico</normaloff>:/image/star.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonNext">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>523</y>
      <width>110</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(155, 255, 200,125);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/image/next.png</normaloff>:/image/next.png</iconset>
    </property>
   </widget>
   <widget class="QTableWidget" name="musicList">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>31</y>
      <width>402</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/musicListBack.png);


</string>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>音乐</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="buttonInit">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>194</y>
      <width>119</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>···</string>
    </property>
   </widget>
   <widget class="QSlider" name="volumSlider">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>22</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labVolum">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonVolum">
    <property name="geometry">
     <rect>
      <x>437</x>
      <y>492</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/image/volum.png</normaloff>:/image/volum.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonhelp">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>523</y>
      <width>35</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(234, 255, 201,100);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/image/help.png</normaloff>:/image/help.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labhand">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/image/hand.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>22</number>
    </property>
   </widget>
   <widget class="QSlider" name="positiSlider">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>500</y>
      <width>400</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonreload">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>75</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 101, 125);
border-image: url(:/image/reload.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonback">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 135, 99, 125);
border-image: url(:/image/back.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonadd">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>111</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/image/add.png);
background-color: rgba(179, 255, 131, 125);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMode">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>523</y>
      <width>45</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(169, 255, 29, 200);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/image/loop.png</normaloff>:/image/loop.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="labhelp">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>200</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(192, 255, 73, 145);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;左键双击音乐可直接播放&lt;/p&gt;&lt;p&gt;快捷键：&lt;/p&gt;&lt;p&gt;下一首：ctrl+E\n&lt;/p&gt;&lt;p&gt;上一首：ctrl+Q\n&lt;/p&gt;&lt;p&gt;大声点儿：ctrl+W\n&lt;/p&gt;&lt;p&gt;小声点儿：ctrl+S\n&lt;/p&gt;&lt;p&gt;播放/暂停：ctrl+F\n&lt;/p&gt;&lt;p&gt;重载音乐列表：ctrl+R\n&lt;/p&gt;&lt;p&gt;添加音乐列表：ctrl+C&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
